src/emailagent.cpp

```diff
@@ -1160,6 +1160,14 @@ bool EmailAgent::easCalendarInvitationResponse(const QMailMessage &message,
         return false;
     }
 
+    if (response != InvitationResponseAccept
```

Comment:
This check being up here prevents invalid responses from being saved, but I suppose that's a favourable change.